app/controllers/admin/suppliers/delivery_partners_controller.rb

```diff
@@ -3,10 +3,10 @@
 module Admin
   module Suppliers
     class DeliveryPartnersController < Admin::BaseController
-      before_action :show, only: %i[show_users show_lps show_schools]
-
+      before_action :set_delivery_partner, only: %i[show delete edit update destroy]
       skip_after_action :verify_policy_scoped
 
+      # region create
```

Comment:
I don't get this comment

Reply:
Probably means the controller is too big :P

Reply:
Probably adding IDE specific comments in the codebase isn't a good idea.

Reply:
They're pretty widely supported https://code.visualstudio.com/updates/v1_17#_folding-regions

Reply:
Happy to remove if you feel strongly though

Reply:
It even works for vim 😛 https://www.cs.cornell.edu/courses/cs6120/2019fa/blog/vim-syntax-highlighting/#:~:text=Vim%20provides%20one%20final%20level,synGroup1%2CsynGroup2%2C...

Reply:
> It even works for vim 😛

Not by default, from what I can tell!

Reply:
Hehe, does vim require a lot of configuration to make it any good? Well yes, weeks of it ;)

I think there is a larger issue here, and the issue is "our controller is bloated because it serves both standard edit / delete actions and a multi-page flow from before". Maybe it's worth thinking about having separate controllers for the two? Maybe not in this PR, maybe as a tech debt ticket.

Reply:
I'd rather you took them out. What does the team think? @DFE-Digital/ecf

Reply:
My opinion is: we need 2 controllers for it, done in a later tech-debt PR.